ietf/static/js/datepicker.js

```diff
@@ -1 +1,2039 @@
-import "bootstrap-datepicker/js/bootstrap-datepicker.js";
```

Comment:
This diff seems to pull in the entire source, rather than include by reference?

Reply:
Yes, as it's been causing issues in some scenarios.

Because that package is being fetched directly from GitHub, it uses ssh when cloning the repo. That process sometime fails because of incorrect ssh settings on the host.

There's also the issue that there's no versioning as it always fetch whatever is currently on GitHub. A change there could easily break the build.

So while not ideal, including the source directly is the best option for now to address both issues. These files will eventually go away once we get rid of jquery.

Reply:
We are planning to get rid of jquery?

Reply:
At some point yes, as pages get refactored to more modern frameworks like Vue.js.

There's really no reason nowadays to use jquery. All browsers implement the querySelector methods. In fact, Bootstrap 5 no longer relies on jquery either.

Other bits that we currently use like scroll to element and scroll spy have native implementations now via scrollIntoView() and the IntersectionObserver class. Both offers much better performance than their jquery counterparts.